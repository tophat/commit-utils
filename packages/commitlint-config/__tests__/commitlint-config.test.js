'use strict'
const { rules } = require('../src')

describe('commitlint-config', () => {
    it('config matches snapshot', () => {
        expect(rules).toMatchSnapshot()
    })
})
