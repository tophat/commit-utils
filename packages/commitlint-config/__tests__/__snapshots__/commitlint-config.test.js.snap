// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`commitlint-config config matches snapshot 1`] = `
Object {
  "body-leading-blank": Array [
    1,
    "always",
  ],
  "footer-leading-blank": Array [
    1,
    "always",
  ],
  "header-max-length": Array [
    2,
    "always",
    72,
  ],
  "subject-empty": Array [
    2,
    "never",
  ],
  "type-case": Array [
    2,
    "always",
    "lower-case",
  ],
  "type-empty": Array [
    2,
    "never",
  ],
  "type-enum": Array [
    2,
    "always",
    Array [
      "wip",
      "feat",
      "fix",
      "cr",
      "style",
      "refactor",
      "perf",
      "docs",
      "test",
      "revert",
      "build",
      "ci",
      "chore",
    ],
  ],
}
`;
